apis/orgs/v1alpha1/groupversion_info.go

```diff
@@ -14,9 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha1 contains the v1alpha1 group Sample resources of the Template provider.
+// Package v1alpha1 contains the core resources of the InfluxDB provider.
 // +kubebuilder:object:generate=true
-// +groupName=sample.influxdb.crossplane.io
+// +groupName=orgs.influxdb.crossplane.io
```

Comment:
I'm not sure about that one, whether I should use `orgs` subgroup or just add all to `influxdb.crossplane.io`. `orgs` appears in URL [as parent ](https://docs.influxdata.com/influxdb/v2.1/api/#operation/PostOrgsIDMembers) for `Organization`, `OrganizationMembership` and `OrganizationOwnership` but that's pretty much it. For `Bucket` it's even less and for `User`, there isn't anything else. So, it all could possibly go under `influxdb.crossplane.io` since there is no explicit grouping really.